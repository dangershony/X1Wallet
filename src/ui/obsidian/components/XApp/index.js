import XAppComponent from './XAppComponent'

export default XAppComponent
