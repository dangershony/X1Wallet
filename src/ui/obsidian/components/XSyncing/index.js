import XSyncingComponent from './XSyncingComponent'

export XAddressComponent from './XAddressComponent'
export XNewWalletHeaderComponent from './XNewWalletHeaderComponent'
export XOldWalletHeaderComponent from './XOldWalletHeaderComponent'
export XProgressComponent from './XProgressComponent'
export XTutorialsComponent from './XTutorialsComponent'

export default XSyncingComponent
