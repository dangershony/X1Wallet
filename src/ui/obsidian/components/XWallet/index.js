import XWalletComponent from './XWalletComponent'

export default XWalletComponent
