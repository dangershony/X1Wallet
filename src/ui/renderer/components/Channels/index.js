import Channels from './Channels'

export default Channels
export ChannelBackButton from './ChannelBackButton'
export ChannelBalance from './ChannelBalance'
export ChannelCapacity from './ChannelCapacity'
export ChannelCardList from './ChannelCardList'
export ChannelCardListItem from './ChannelCardListItem'
export ChannelCloseDialog from './ChannelCloseDialog'
export ChannelCount from './ChannelCount'
export ChannelCreate from './ChannelCreate'
export ChannelCreateForm from './ChannelCreateForm'
export ChannelCreateHeader from './ChannelCreateHeader'
export ChannelCreateSummary from './ChannelCreateSummary'
export ChannelData from './ChannelData'
export ChannelDetail from './ChannelDetail'
export ChannelFilter from './ChannelFilter'
export ChannelFooter from './ChannelFooter'
export ChannelHeader from './ChannelHeader'
export ChannelMoreButton from './ChannelMoreButton'
export ChannelNodeSearch from './ChannelNodeSearch'
export ChannelSearch from './ChannelSearch'
export ChannelStatus from './ChannelStatus'
export ChannelSummaryList from './ChannelSummaryList'
export ChannelSummaryListItem from './ChannelSummaryListItem'
export ChannelsActions from './ChannelsActions'
export ChannelsCapacity from './ChannelsCapacity'
export ChannelsCapacityDonut from './ChannelsCapacityDonut'
export ChannelsHeader from './ChannelsHeader'
export ChannelsInfo from './ChannelsInfo'
export ChannelsRefresh from './ChannelsRefresh'
export ChannelsSummaryDonut from './ChannelsSummaryDonut'
export ChannelsViewSwitcher from './ChannelsViewSwitcher'
export NodeCardList from './NodeCardList'
export NodeCardView from './NodeCardView'
