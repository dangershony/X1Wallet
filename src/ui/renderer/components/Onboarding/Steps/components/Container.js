import styled from 'styled-components'
import { Flex } from 'rebass'

const Container = styled(Flex)`
  position: absolute;
  top: 0;
  bottom: 0;
`

export default Container
