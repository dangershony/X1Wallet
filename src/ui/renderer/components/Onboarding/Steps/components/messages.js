import { defineMessages } from 'react-intl'

/* eslint-disable max-len */
export default defineMessages({
  skip_backup_dialog_warning:
    'Not setting up your channel backup means the file will only live locally in your Zap data directory. If this file is damaged or lost, it can lead to loss of funds in the event that you need to restore from seed.',
  skip_backup_dialog_restore_warning:
    'Recovering a wallet without your channel backup file can lead to loss of funds in the event that you need to restore from seed.',
  skip_backup_dialog_warning_skip_text: 'Skip',
  skip_backup_dialog_warning_cancel_text: 'Cancel',
  skip_backup_dialog_warning_header: 'Are you sure?',
  skip_backup_dialog_warning_acknowledgement: 'I do not want to set up my channel backup',
  skip_backup_dialog_restore_warning_acknowledgement: 'I do not want to recover channel funds',
  error_dialog_header: 'An error has occurred',
  error_dialog_create_wallet_error_desc: 'The following error has ocurred when creating a wallet:',
  error_dialog_recover_wallet_error_desc:
    'The following error has occurred when recovering a wallet:',
  error_dialog_close_text: 'Close',
})
