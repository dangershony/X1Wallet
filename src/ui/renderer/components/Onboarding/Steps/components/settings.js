export const BACKUP_FORM_WIDTH = 630
export const BACKUP_FORM_HEIGHT = 470
