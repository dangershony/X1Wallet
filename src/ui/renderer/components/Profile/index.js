export ProfileMenu from './ProfileMenu'
export ProfilePage from './ProfilePage'
export ProfilePaneConnect from './ProfilePaneConnect'
export ProfilePaneNodeInfo from './ProfilePaneNodeInfo'
