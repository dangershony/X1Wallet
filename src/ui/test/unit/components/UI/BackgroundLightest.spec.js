import React from 'react'
import renderer from 'react-test-renderer'
import { BackgroundSecondary } from 'components/UI'

describe('component.UI.BackgroundSecondary', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<BackgroundSecondary />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
