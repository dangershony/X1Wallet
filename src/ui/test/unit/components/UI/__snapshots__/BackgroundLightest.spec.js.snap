// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.BackgroundSecondary should render correctly 1`] = `
.c0 {
  box-sizing: border-box;
  color: primaryText;
  background-color: secondaryColor;
}

<div
  className="c0"
  color="primaryText"
/>
`;
